src/python/deepgnn/pytorch/common/utils.py

```diff
@@ -13,19 +13,18 @@
 
 from deepgnn import get_logger
 from deepgnn.pytorch.common.consts import PREFIX_CHECKPOINT
-from deepgnn.graph_engine import FeatureType
 
 
-def get_feature_type(feature_type_str: str):
+def get_dtype(dtype_str: str):
```

Comment:
Can we get rid of this function completely and parse arguments with a plain `np.dtype` constructor?

Reply:
It converts form np.dtype to python type - I cannot remove its use case. And the function is a bit complex: type(dtype(0).item()), maybe better to just have the helper function.